templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Are you sure to delete the car: {{ car }}</p>
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+  <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.